pkg/roachprod/opentelemetry/files/opentelemetry-config.yaml.tmpl

```diff
@@ -0,0 +1,122 @@
+---
+extensions:
+  health_check:
+
+receivers:
+  {{- if .CockroachDBMetrics }}
+  prometheus/cockroachdb:
+    config:
+      global:
+        scrape_interval: 30s
+      scrape_configs:
+      - job_name: cockroachdb
+        honor_timestamps: false
+        metrics_path: /_status/vars
+        tls_config:
+          insecure_skip_verify: true
+        follow_redirects: true
+        enable_http2: true
+        static_configs:
+          # Multiple targets are listed to cover the cases when the default
+          # HTTP port is used or when scanning for a free HTTP port starting
+          # from 29000. In the latter case 29000 will be used for SQL and 29001
+          # for HTTP.
+          - targets:
+            - "localhost:26258"
+            - "localhost:29001"
+  {{- end }}
+  hostmetrics:
+    collection_interval: 10s
+    scrapers:
+      paging:
+        metrics:
+          system.paging.utilization:
+            enabled: true
+      cpu:
+        metrics:
+          system.cpu.utilization:
+            enabled: true
+      disk:
+      filesystem:
+        metrics:
+          system.filesystem.utilization:
+            enabled: true
+      load:
+      memory:
+      network:
+      processes:
+
+processors:
+  batch/datadog:
+    send_batch_max_size: 1000
+    send_batch_size: 100
+    timeout: 10s
+  memory_limiter/datadog:
```

Comment:
Nit: any particular reason for this specific limit? Seems reasonable to me.

Reply:
Nope! I just took the values from https://docs.datadoghq.com/opentelemetry/collector_exporter/collector_batch_memory and increased the memory limit from 1000 to 1024 because binary.